whois-api/src/main/java/net/ripe/db/whois/api/UpdatesParser.java

```diff
@@ -18,7 +18,7 @@
 import net.ripe.db.whois.update.domain.UpdateContext;
 import net.ripe.db.whois.update.domain.UpdateMessages;
 import net.ripe.db.whois.update.keycert.PgpSignedMessage;
-import net.ripe.db.whois.update.keycert.X509CertificateWrapper;
+import net.ripe.db.whois.common.clientauthcertificates.X509CertificateWrapper;
```

Comment:
The package name is a bit unwieldly, can you rename to something shorter like `clientcert` or `certauth` or `x509` ?